getting-started/bookshelf/db_memory.go

```diff
@@ -27,43 +29,45 @@ var _ BookDatabase = &memoryDB{}
 // memoryDB is a simple in-memory persistence layer for books.
 type memoryDB struct {
 	mu     sync.Mutex
-	nextID int64           // next ID to assign to a book.
-	books  map[int64]*Book // maps from Book ID to Book.
+	nextID int64            // next ID to assign to a book.
+	books  map[string]*Book // maps from Book ID to Book.
 }
 
 func newMemoryDB() *memoryDB {
 	return &memoryDB{
-		books:  make(map[int64]*Book),
+		books:  make(map[string]*Book),
 		nextID: 1,
 	}
 }
 
 // Close closes the database.
-func (db *memoryDB) Close() {
+func (db *memoryDB) Close(ctx context.Context) error {
```

Comment:
The parameter `ctx` is unused, so it'd be slightly better to leave it out, as you have already done in the `firestoreDB.Close` method.

Reply:
Done.